### PR TITLE
Add few doltpy commands that broke with dolt compatibility changes

### PR DESCRIPTION
**Pull request by max-hoffman**
Some bats that if failed indicate a doltpy incompatibility. The tests could be more involved, but the goal here isn't row-output correctness.

This list will grow as I rewrite more dolt commands to use SQL features.

### CODE REVIEW

File: integration-tests/bats/doltpy.bats [hunk 1]
```diff
@@ -0,0 +1,62 @@
...
+}
+
+# NOTE: if these break on a release that's OK, just tell Max/Oscar
+
+# already covered by other tests:
+# "select * from dolt_branches"
+# "SELECT * FROM `{table}` ASOF '{commit}'"
+
+# verifying the output comparison to dolt log would be nice
+@test "doltpy: Dolt.log" {
```
Comment (timsehn) [inline integration-tests/bats/doltpy.bats:29]: I think you should put the assertion in the test name. Like:

"dolt log returns proper named headers"

File: integration-tests/bats/doltpy.bats [hunk 2]
```diff
@@ -0,0 +1,62 @@
...
+    message as message
+FROM
+    dolt_log AS dc
+    LEFT OUTER JOIN dolt_commit_ancestors AS dca
+        ON dc.commit_hash = dca.commit_hash
+    WHERE dc.commit_hash = HASHOF('HEAD')
+    ORDER BY date DESC
+    LIMIT 1;
+SQL
+    echo $status
```
Comment (timsehn) [inline integration-tests/bats/doltpy.bats:46]: Get rid of theses. They are trapped if the test passes and you get output if it fails. echos are useless.

**Review by timsehn — COMMENTED** [review]
Couple comments.

File: integration-tests/bats/doltpy.bats (hunk 2 above)
Comment (max-hoffman) [reply integration-tests/bats/doltpy.bats:46]: fixing

**Comment by timsehn** [comment]
LGTM

**Pull request merged**